src/components/PatientRegistration/PatientRegistration.tsx

```diff
@@ -1,12 +1,15 @@
 import React, { ReactElement, useState, useEffect } from 'react'
 import './patientRegistration.css'
 import Checkbox from './Checkbox';
+import { authApi } from '../../api/authApi';
 
 
 export default function PatientRegistration(): ReactElement {
   const [firstName, setName] = useState('');
   const [lastName, setLastName] = useState('');
-  const [dateBirth, setDateBirth] = useState('');
+
+  // const [startDate, setStartDate] = useState<any>(new Date());
+  const [dateBirth, setDateBirth] = useState<any>(new Date());
```

Comment:
using any in your typings leaves you without the power of typescript. As the default value is new Date, may be would be nice to make <Date> type of useStae?

Reply:
like this

Reply:
const [dateBirth, setDateBirth] = useState<Date>();

or

const [dateBirth, setDateBirth] = useState<>(Date);

Reply:
const [dateBirth, setDateBirth] = useState<Date>();